java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -465,6 +474,16 @@ private void executeMethodInvocation(MethodInvocationTree mit) {
     ProgramState.Pop unstack = programState.unstackValue(mit.arguments().size() + 1);
     programState = unstack.state;
     logState(mit);
+
+    // get method behavior for method with known declaration (ie: within the same file)
+    Tree declaration = mit.symbol().declaration();
+    if(declaration != null) {
+      // (FIXME) nothing done of the invoked behavior
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1134)